types/websocket/index.d.ts

```diff
@@ -1,26 +1,26 @@
-// Type definitions for websocket
+// Type definitions for websocket 1.0
 // Project: https://github.com/theturtle32/WebSocket-Node
 // Definitions by: Paul Loyd <https://github.com/loyd>,
 //                 Kay Schecker <https://github.com/flynetworks>,
 //                 Zhao Lei <https://github.com/zhaoleimxd>
-//                 Sheng Chen <https://github.com/jdneo>
+//                 Sheng Chen <https://github.com/jdneo>,
+//                 Matthew Peveler <https://github.com/MasterOdin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.2
 
 /// <reference types="node" />
 
-
 import events = require('events');
 import http = require('http');
 import https = require('https');
 import net = require('net');
 import url = require('url');
 
-export interface IStringified {
+export interface Stringified {
```

Comment:
I suggest not renaming the interfaces as they introduce a breaking change for people importing them.

Reply:
Makes sense, reverted all interfaces to have the leading `I` again and ignored the appropriate tslint rule.